README.md

```diff
@@ -16,7 +16,7 @@
 
 To use the Swift 3 version, add this to your Podfile (until 2.2 or higher is released):
 ```ruby
-pod 'ChameleonFramework/Swift', :git => 'https://github.com/ViccAlexander/Chameleon.git'
+    pod 'ChameleonFramework/Swift', :git => 'https://github.com/ViccAlexander/Chameleon.git', :branch => 'swift3'
```

Comment:
Master is swift3 compatible. `swift3` branch is an old one, I'll delete it later.